requirements/base.txt

```diff
@@ -66,39 +76,49 @@ django-environ==0.11.2
     # via -r base.in
 django-extensions==3.2.3
     # via -r base.in
-django-ratelimit==2.0.0
+django-ratelimit==4.1.0
```

Comment:
We either have to pin to `<4` or switch how we're importing: https://django-ratelimit.readthedocs.io/en/latest/upgrading.html#from-3-x-to-4-0